### PR TITLE
add few missing meeting options for Android

### PR DESCRIPTION
**Pull request by nadishan**

### CODE REVIEW

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java [hunk 1]
```diff
@@ -168,6 +168,15 @@ public void joinMeeting(
...
       if(paramMap.hasKey("noVideo")) opts.no_video = paramMap.getBoolean("noVideo");
+      if(paramMap.hasKey("noInvite")) opts.no_invite = paramMap.getBoolean("noInvite");
+      if(paramMap.hasKey("noBottomToolbar")) opts.no_bottom_toolbar = paramMap.getBoolean("noBottomToolbar");
+      if(paramMap.hasKey("noPhoneDialIn")) opts.no_dial_in_via_phone = paramMap.getBoolean("noPhoneDialIn");
+      if(paramMap.hasKey("noPhoneDialOut")) opts.no_dial_out_to_phone = paramMap.getBoolean("noPhoneDialOut");
+      if(paramMap.hasKey("noMeetingEndMessage")) opts.no_meeting_end_message = paramMap.getBoolean("noMeetingEndMessage");
+      if(paramMap.hasKey("noMeetingErrorMessage")) opts.no_meeting_error_message = paramMap.getBoolean("noMeetingErrorMessage");
+      if(paramMap.hasKey("noShare")) opts.no_share = paramMap.getBoolean("noShare");
+      if(paramMap.hasKey("noTitlebar")) opts.no_titlebar = paramMap.getBoolean("noTitlebar");
+      if(paramMap.hasKey("noTitlebar")) opts.no_titlebar = paramMap.getBoolean("noTitlebar");
```
Comment (mieszko4) [inline android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java:180]: repeated option
```suggestion
      if(paramMap.hasKey("noTitlebar")) opts.no_titlebar = paramMap.getBoolean("noTitlebar");

```

**Review by mieszko4 — COMMENTED** [review]
Thnx! Please see my comments.

Based on https://marketplace.zoom.us/docs/sdk/native-sdks/iOS/sdk-reference it looks like there are not equivalent fields on ios sdk.

Could you also please update add the new fields in `index.ts` `RNZoomUsJoinMeetingParams`?
Something like:
```ts
// android only fields:
noInvite: boolean;
```
ect.

File: android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java (hunk 1 above)
Comment (nadishan) [reply android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java]: Review comments addressed. Please take a look.

**Review by mieszko4 — APPROVED** [review]
lgtm, thnx

**Comment by amdrade** [comment]
Hi, thanks for package!
To `noInvite` fields, can use:
`[[MobileRTC sharedRTC] getMeetingSettings].meetingInviteHidden = YES;`
I changed the file directly and it worked, adding in the joinMeeting.
[Hide invitation link option in Swift](https://devforum.zoom.us/t/hide-invitation-link-option-in-swift/37699)

**Comment by mieszko4** [comment]
> Hi, thanks for package! To `noInvite` fields, can use: `[[MobileRTC sharedRTC] getMeetingSettings].meetingInviteHidden = YES;` I changed the file directly and it worked, adding in the joinMeeting. [Hide invitation link option in Swift](https://devforum.zoom.us/t/hide-invitation-link-option-in-swift/37699)

Thnx @amdrade! Feel free to make a PR :)

**Comment by rahulvyas** [comment]
> Hi, thanks for package! To `noInvite` fields, can use: `[[MobileRTC sharedRTC] getMeetingSettings].meetingInviteHidden = YES;` I changed the file directly and it worked, adding in the joinMeeting. [Hide invitation link option in Swift](https://devforum.zoom.us/t/hide-invitation-link-option-in-swift/37699)

How to do this in react native

**Comment by mieszko4** [comment]
You would need do native iOS change in `RNZoomUs.m` in `joinMeeting` before calling `joinMeetingWithJoinParam`, something like:
```objective-c
[[MobileRTC sharedRTC] getMeetingSettings].meetingInviteHidden = data[@"noInvite"];
```

**Pull request merged**